### PR TITLE
REFACTOR: Primitive scale

### PR DESCRIPTION
**Pull request by svandenb-dev**
Moving scale to EDBPrimitive to be consistent with expand method.

### CODE REVIEW

**Review by maxcapodi78 — APPROVED** [review]

File: src/pyedb/dotnet/edb_core/dotnet/primitive.py [hunk 1]
```diff
@@ -376,6 +376,39 @@ def expand(self, offset=0.001, tolerance=1e-12, round_corners=True, maximum_corn
...
+        -------
+        bool
+           ``True`` when successful, ``False`` when failed.
+        """
+        if not isinstance(factor, str):
+            factor = float(factor)
+            polygon_data = self.polygon_data.create_from_arcs(self.polygon_data.edb_api.GetArcData(), True)
+            if not center:
+                center = self.polygon_data.edb_api.GetBoundingCircleCenter()
+                if center:
```
Comment (SMoraisAnsys) [inline src/pyedb/dotnet/edb_core/dotnet/primitive.py:399]: Can you log an error when this if statement is not valid ? Or assume that it should work (I mean pass the if statement) and remove this condition ?

**Review by SMoraisAnsys — CHANGES_REQUESTED** [review]
Why are you adding an empty file to the PR ?
I left a comment on the scale function.

File: src/pyedb/dotnet/edb_core/dotnet/primitive.py (hunk 1 above)
Comment (svandenb-dev) [reply src/pyedb/dotnet/edb_core/dotnet/primitive.py:399]: Done

**Review by ring630 — APPROVED** [review]
LGTM

**Review by SMoraisAnsys — COMMENTED** [review]
@svandenb-dev Please remove the empty file as asked in my previous review

**Review by ring630 — COMMENTED** [review]
Please fix the empty file.

**Review by ring630 — APPROVED** [review]

**Pull request merged**